src/lib/contentful/fragments/ComponentRoutingItemFragment.ts

```diff
@@ -0,0 +1,31 @@
+import { graphql } from "gql.tada";
+
+export const ComponentRoutingItemFragment = graphql(`
+  fragment ComponentRoutingItemFragment on ComponentRoutingItem {
+    __typename
+    sys {
+      id
+    }
+    contentfulLabel
+    heading
+    eyebrowText
+    subheadingText
+    description
+    link {
+      ... on ComponentLink {
```

Comment:
We have a `LinkFragment` that creates the standard for how `ComponentLink` and the actual component `<Link>` are resolved.